main.go

```diff
@@ -101,20 +102,34 @@ func main() {
 	setStaticMetrics()
 	if *flagMetricsAddress != "" {
 		server := tgo.NewMetricServer()
-		address := *flagMetricsAddress
-
-		if !strings.Contains(address, ":") {
-			if !tstrings.IsInt(address) {
-				fmt.Printf("Metrics address must be of the form \"host:port\" or \":port\" or \"port\".\n")
-				return
-			}
-			address = ":" + address
+		address, err := parseAddress(*flagMetricsAddress)
+		if err != nil {
+			fmt.Printf("%s", err)
+			return
 		}
-
 		go server.Start(address)
 		defer server.Stop()
 	}
 
+	// Health Check endpoint
+
+	if *flagHealthCheck != "" {
+		address, err := parseAddress(*flagHealthCheck)
+		if err != nil {
+			fmt.Printf("%s", err)
+			return
+		}
+		thealthcheck.Configure(address)
+
+		go thealthcheck.Start()
+		defer thealthcheck.Stop()
+
+		// Add a static "ping" endpoint
+		thealthcheck.AddEndpoint("/ping", func()(code int, body string){
```

Comment:
If we stick to "reserved terms are of the format \_*\_" this should be /\_PING\_ ... just a thought